README.md

```diff
@@ -1,113 +1,167 @@
+
+### Write
+*   😌 Simple collaborative editing without the formatting complexity of markdown
+*   🔌 Offline? No problem, keep writing, your edits will get synced when back online
+*   💅 Clean formatting options to focus on content creation and create beautiful documents
+*   🧱 Built for productivity (markdown support, many block types, slash commands, markdown support, keyboard shortcuts)
+*   🤖 Save time thanks to our AI actions (generate, sum up, correct, translate)
+
+### Collaborate
+*   🤝 Collaborate in realtime with your team mates
+*   🔒 Granular access control to keep your information secure and shared with the right people
+*   📑 Professional document exports in multiple formats (.odt, .doc, .pdf) with customizable templates
+*   📚 Built-in wiki functionality to transform your team's collaborative work into organized knowledge `ETA 02/2025`
+
+### Self-host
+*   🚀 Easy to install, scalable (Kube first) and secure alternative to Notion, Outline and Confluence
+
+## Getting started 🔧
+### Run it locally
+**Prerequisite**
+Make sure you have a recent version of Docker and [Docker Compose](https://docs.docker.com/compose/install) installed on your laptop:
+
+```shellscript
 $ docker -v
-  Docker version 20.10.2, build 2291f61
+
+Docker version 20.10.2, build 2291f61
 
 $ docker compose -v
-  docker compose version 1.27.4, build 40524192
-```
 
-> ⚠️ You may need to run the following commands with `sudo` but this can be
-> avoided by assigning your user to the `docker` group.
+docker compose version 1.27.4, build 40524192
+```
 
-### Project bootstrap
+> ⚠️ You may need to run the following commands with sudo but this can be avoided by assigning your user to the `docker` group.
 
+**Project bootstrap**
 The easiest way to start working on the project is to use GNU Make:
 
-```bash
+```shellscript
 $ make bootstrap FLUSH_ARGS='--no-input'
 ```
 
-This command builds the `app` container, installs dependencies, performs
-database migrations and compile translations. It's a good idea to use this
-command each time you are pulling code from the project repository to avoid
-dependency-releated or migration-releated issues.
+This command builds the `app` container, installs dependencies, performs database migrations and compile translations. It's a good idea to use this
+
+command each time you are pulling code from the project repository to avoid dependency-releated or migration-releated issues.
 
 Your Docker services should now be up and running 🎉
 
-You can access to the project by going to http://localhost:3000.
+You can access to the project by going to <http://localhost:3000>.
+
 You will be prompted to log in, the default credentials are:
-```bash
+
+```shellscript
 username: impress
+
 password: impress
 ```
 
 📝 Note that if you need to run them afterwards, you can use the eponym Make rule:
 
-```bash
+```shellscript
 $ make run-with-frontend
```

Comment:
all the front-end part of this was broken. I think @lunika can help making it work again with just a "make bootstrap".
A demo site for non developers would be nice?

Reply:
Yeah, down the road I'd like to have a dedicated demo env. 
I've checked with Arnaud, it triggered a question around the authentication. Do we want to setup a keycloak just for this ? 
Meanwhile we decided to add a link to the preprod environments and provide people with a yopmail account.